### PR TITLE
Story/cloud21

### PR DESCRIPTION
**Pull request by birgitlewicki**

### CODE REVIEW

File: src/modules/orders/infrastructure/repositories/avvcatalogue.repository.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import { AbstractRepository } from '../../../shared/infrastructure';
+import { AVVCatalogueObject } from '../../../shared/infrastructure/parse-types';
+import { AVVCatalog } from '../../legacy/model/avvcatalog.entity';
+import {
+    AVVCatalogData,
+    AVVCatalogObject
+} from '../../legacy/model/legacy.model';
```
Comment (dptolle) [inline src/modules/orders/infrastructure/repositories/avvcatalogue.repository.ts:7]: Please no dependencies to legacy code

File: src/modules/shared/infrastructure/cache/avvcatalogue.cache.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import { AVVCatalogueNameData } from '../../../orders/infrastructure/repositories/avvcatalogue.repository';
+import { AVVCatalog } from '../../../orders/legacy/model/avvcatalog.entity';
```
Comment (dptolle) [inline src/modules/shared/infrastructure/cache/avvcatalogue.cache.ts:2]: Please no dependencies to legacy code

**Review by dptolle — CHANGES_REQUESTED** [review]

File: src/modules/orders/infrastructure/repositories/avvcatalogue.repository.ts (hunk 1 above)
Comment (birgitlewicki) [reply src/modules/orders/infrastructure/repositories/avvcatalogue.repository.ts]: done!

File: src/modules/shared/infrastructure/cache/avvcatalogue.cache.ts (hunk 2 above)
Comment (birgitlewicki) [reply src/modules/shared/infrastructure/cache/avvcatalogue.cache.ts]: done!

**Pull request merged**